scripts/checkpatch.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
```

Comment:
``#!/usr/bin/env perl -w`` is not valid.

Change this to ``#!/usr/bin/env perl`` and add ``use warnings;`` on the first script execution line.